PyRDF/__init__.py

```diff
@@ -81,7 +73,7 @@ def _get_paths_set_from_string(path_string):
         set: The set with all paths returned from the directory, or a set
             with only the path of the string.
     """
-    logger.debug("Retrieving paths from {}".format(path_string))
+    logger.debug("Retrieving paths from %s", path_string)
```

Comment:
So all this will be `RLogger` in another PR, right?

Reply:
RLogger is quite new so first I have to understand it better. As Philippe said in the meeting it would be nice to have the simplicity of the `logging` module in RLogger. But yes, in the future when PyRDF will be in ROOT probably I'll change to that